docs/topics/enum-classes.md

```diff
@@ -103,9 +115,45 @@ inline fun <reified T : Enum<T>> printAllValues() {
 printAllValues<RGB>() // prints RED, GREEN, BLUE
 ```
 
-Every enum constant has properties for obtaining its name and position (starting with 0) in the enum class declaration:
+> For more information about inline functions and reified type parameters, see [Inline functions](inline-functions.md).
+> 
+> {type="tip"}
+
+In Kotlin 1.8.20, the `entries` property is introduced as a future replacement for the `values()` function. The 
+`entries` property returns a pre-allocated immutable list of your enum constants. This is particularly useful when you 
+are working with [collections](collections-overview.md) and can you help you avoid [performance issues](https://github.com/Kotlin/KEEP/blob/master/proposals/enum-entries.md#examples-of-performance-issues).
+
+For example:
+```kotlin
+enum class RGB { RED, GREEN, BLUE }
+
+@OptIn(ExperimentalStdlibApi::class)
+fun main() {
+    for (color in RGB.entries) println(color.toString())
+    // prints RED, GREEN, BLUE
+}
+```
+
+> The `entries` property is Experimental. To use it you need to opt in by using `@OptIn(ExperimentalStdlibApi)`, and
```

Comment:
Hi @danil-pavlov, originally I had this link but it was removed because Mikhail requested it in his review. Please see [his comment](https://github.com/JetBrains/kotlin-web-site/pull/3430#discussion_r1113070125) for further details. With this in mind, is it OK to not have the link?

Reply:
OK, sure. But I'd still get rid of the "uset it.. by using repetition"

Reply:
@danil-pavlov, so is this what you think it should look like?
```
> The `entries` property is Experimental. To use it you need to
> [set the language version to 1.9](gradle-compiler-options.md#attributes-common-to-jvm-and-js).
>
{type="warning"}
```

Reply:
Is opt-in required too? I'd also replace modal verb with imperative, but it's up to you of course

> The `entries` property is Experimental. To use it, opt in with `@...` and 
> [set the language version to 1.9](gradle-compiler-options.md#attributes-common-to-jvm-and-js).
>
{type="warning"}